arches/app/etl_modules/tile_excel_exporter.py

```diff
@@ -144,6 +145,11 @@ def run_load_task_async(self, request):
         resource_ids = request.POST.get("resource_ids", None)
         export_concepts_as = request.POST.get("export_concepts_as")
         filename = request.POST.get("filename")
+        search_url = request.POST.get("search_url", None)
+        if search_url is not None and resource_ids is None:
```

Comment:
Unless there is a particular reason for a None check, I think this could simply be: 
```
if search_url and not resource_ids
```